### PR TITLE
Use localVerifier for SD tests.

### PR DESCRIPTION
**Pull request by tminard**

### CODE REVIEW

File: tests/40-sd-create.js [hunk 1]
```diff
@@ -127,14 +130,33 @@ describe('ecdsa-sd-2023 (create)', function() {
...
+                  // disclose only the `credentialSubject.id` path
+                  cryptosuite: ecdsaSd2023Cryptosuite
+                    .createDiscloseCryptosuite({
+                      selectivePointers: [
+                        '/credentialSubject/id'
+                      ]
+                    })
+                });
+
+                const derivedCredential = await jsigs.derive(issuedVc, {
```
Comment (aljones15) [inline tests/40-sd-create.js:150]: there is an existing deriveCredential function in `vc-generator` used for the sd-verify test data.

**Review by aljones15 — CHANGES_REQUESTED** [review]
the vc-generator's derive stuff should handle the derivation and remove some of these new imports in this file. Aside from that good work and really impressed with this.

File: tests/40-sd-create.js (hunk 1 above)
Comment (aljones15) [reply tests/40-sd-create.js:150]: also might want to add a comment about how anyone can derive a proof from a base proof.

File: tests/40-sd-create.js [hunk 2]
```diff
@@ -127,14 +130,33 @@ describe('ecdsa-sd-2023 (create)', function() {
                 '"proofValue" property that starts with "u".'
               );
             });
-            it('The "proof" MUST verify when using a conformant verifier.',
+            it('The "proof" MUST verify with a conformant verifier.',
```
[the remark this answers is not in the document]
Comment (tminard) [reply tests/40-sd-create.js:133]: No I do not see any specific normative statements that would mandate this test: https://w3c.github.io/vc-di-ecdsa/#add-derived-proof-ecdsa-sd-2023

Perhaps it is safe to delete?

It seems more like an integration test that is helpful to verify your specific implementation produces a "valid" credential, but it does not have a link in the spec that I could find.

**Review by aljones15 — APPROVED** [review]
Let's get this and finish off the ECDSA stuff.

**Pull request merged**